sass-sys/build.rs

```diff
@@ -49,8 +51,13 @@ fn compile() {
 #[cfg(target_env="msvc")]
 fn compile() {
     let src = get_libsass_folder();
-    let r = Command::new("msbuild")
-        .args(&["win\\libsass.sln", "/p:LIBSASS_STATIC_LIB=1", "/p:Configuration=Release"])
+    let target = env::var("TARGET").expect("TARGET not found in environment");
+    let msvc_platform = if target.contains("x86_64") { "Win64" } else { "Win32" };
+    let r = gcc::windows_registry::find(target.as_str(),"msbuild.exe")
+        .expect("could not find msbuild")
+        .args(&["win\\libsass.sln", "/p:LIBSASS_STATIC_LIB=1", "/p:Configuration=Release",
+            "/p:WholeProgramOptimization=false",
```

Comment:
Unfortunately, it would not link unless I disabled Whole Program Optimization - the linker would say it was going to restart with link time code generation enabled, and then crash.